content/en/blog/2022-05-05-introducing-kluctl-and-flux.md

```diff
@@ -0,0 +1,83 @@
+that Kluctl internally uses Kustomize and Helm extensively.
+
+This might raise the question: Why not use plain Kustomize and/or Helm if Flux is already involved? There are multiple
+reasons to prefer Kluctl, which we'll try to elaborate here:
+
+## Kluctl Projects/Deployments
+If you prefer the way Kluctl organizes and structures projects and deployments, then using the Flux Kluctl Controller
+is obviously the best choice. Kluctl allows you to easily glue together what belongs together. If for example, a redis
+database is required to make your application work, you can manage the redis Helm Release and your application in the
+same deployment, including the necessary configuration to let them talk to each other.
+
+To see how different a Kluctl deployment is compared to classic Kustomize/Helm + Flux, you can compare the
+[flux2-kustomize-helm-example](https://github.com/fluxcd/flux2-kustomize-helm-example) and the
+[Kluctl Microservices Demo](https://github.com/kluctl/kluctl-examples/tree/main/microservices-demo/3-templating-and-multi-env)
+([here]({{< ref "docs/guides/tutorials/microservices-demo">}}) is tutorial for the demo).
+
+## Native multi-env support
+Kluctl allows you to natively create deployment projects that can be deployed multiple times to different
+environments/targets. You can for example have one target that is solely meant for `local` (e.g. Kind based) deployments,
+one that targets the `test` environment and one for `prod`. You can then use templating to influence deployments in whatever
+way you like. For example, you could change the `local` target to set all replicas to 1 and omit resource hungry
+support applications.
+
+This is possible in plain Kustomize as well, but requires you to solve it without the concept of
+[targets]({{< ref "docs/reference/kluctl-project/targets" >}}) and without templating. In Kustomize, multi-env
+deployments must be solved with [overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays),
+which does not necessary align with how you prefer your project structure.
+
+## Mix local DevOps and GitOps
+The core idea of GitOps is that Git becomes the single source of truth for the desired cluster state. This is something
+that is extremely valuable with many advantages compared to other approaches. There are however still situations
+where diverging from GitOps is very valuable as well.
+
+For example, when you start a new deployment project, you're usually in a state of frequent changes inside the deployment
+project. These frequent changes need frequent deployments and testing until you get to a point where things are stable
+enough. If you're forced to adhere to GitOps in that situation, you end up with very noisy Git histories and plenty
+of trial-and-error deployment cycles. This is, at least in our opinion and experience, a major productivity killer
```

Comment:
This is a major productivity killer and we believe there has to be a better way.